data.json

```diff
@@ -1,6 +1,6 @@
 {
   "Go": {
-    "Supported": ["go1.11.5", "go1.12"],
+    "Supported": ["go1.12","go1.11.5","go1.11.4", "go1.11.3", "go1.11.2", "go1.11.1"],
```

Comment:
`go1.11` needs to be added here if we're supporting all minor versions of the last two major releases.